### ISSUE TITLE
Default color scheme doesn't go well with Eclipse dark theme.

### ISSUE THREAD

**Issue by ravijoshi2** [vaulttec/veloedit#15]
Default color scheme doesn't go well with Eclipse dark theme. Any way we can change default color theme of Veloedit?

**Comment by ChristopherSchultz** [comment]
Yes, it's practically unreadable because Eclipse changes only the background color from white to very dark gray and doesn't update the other colors. So you have e.g. `$foo` in blue-on-dark-gray which just looks like a fuzzy mess unless you really strain your eyes.

Is there any way to customize the syntax-highlighting colors... or even disable them entirely? VeloEdit is still useful even without the syntax highlighting (though less so, of course).

**Comment by tjuerge** [comment]
Sure, the default color scheme is not usable with dark scheme. But is there any issue manually selecting different colors in veloedit's editor preferences?

<img width="1279" alt="Screenshot 2020-01-07 at 17 06 02" src="https://user-images.githubusercontent.com/2626998/71909724-e1a4e600-3170-11ea-895b-5a202053715e.png">

**Comment by ChristopherSchultz** [comment]
That works great. I didn't know the colors were tweakable from the prefs UI. Thanks for the pointers.

**Comment by holle** [comment]
I initially had the *same* problem as the original poster of this ticket.
And yes, the shown solution works for me as well....

But for some weird reason the dialog shown in the screenshot above (Velocity -> Editor -> Syntax Highlighting ...) *cannot* be found using the quick search feature in preferences.

Edit: searching for "Editor" yields the preferences dialog but searching for "Syntax", "Highlighting" or "Coloring" does not yield the page, but pretty much all other editors pages are listed ...

Maybe you could fix that and prevent these type of questions?

Thanks in advance ;-)

**Comment by mikekonikoff** [comment]
Updating the editor preferences for syntax highlighting in veloedit does not have any effect when using MyEclipse 2021 DevStyle themes. Under eclipse dark theme, after applying the changes, the editor window has to be reopened for the changes to appear.
